cpp/ql/src/semmle/code/cpp/models/implementations/Strings.qll

```diff
@@ -0,0 +1,27 @@
+import semmle.code.cpp.models.interfaces.Taint
+
+/**
+ * The `std::basic_string` constructor(s).
+ */
+class StringConstructor extends TaintFunction {
```

Comment:
Since the term "string" means so many different things in C/C++, I'd prefer more precise names for this file and its classes: `StdString.qll`, `StdStringConstructor`, `StdStringCStr`.